### PR TITLE
Remove Both member from ProviderType

### PR DESCRIPTION
**Pull request by gunndabad**

### CODE REVIEW

**Review by jameslawesynergy — APPROVED** [review]

File: src/Dfc.CourseDirectory.Core/Models/ProviderType.cs [hunk 1]
```diff
@@ -19,7 +18,7 @@ providerType switch
                 ProviderType.Undefined => "",
                 ProviderType.Apprenticeships => "Apprenticeships",
                 ProviderType.FE => "F.E.",
-                ProviderType.Both => "Both",
+                ProviderType.Apprenticeships | ProviderType.FE => "Both",
```
[the remark this answers is not in the document]
Comment (gunndabad) [reply src/Dfc.CourseDirectory.Core/Models/ProviderType.cs:21]: It's value equality so works fine

File: src/Dfc.CourseDirectory.Web/Views/Provider/UpdateProviderType.cshtml [hunk 2]
```diff
@@ -39,9 +39,9 @@
                                 </label>
                             </div>
                             <div class="govuk-radios__item">
-                                @Html.RadioButtonFor(m => m.ProviderType, ProviderType.Both, new { @class = "govuk-radios__input", id = ProviderType.Both })
-                                <label class="govuk-label govuk-radios__label" for="@ProviderType.Both">
-                                    @WebHelper.GetEnumDescription(ProviderType.Both)
+                                @Html.RadioButtonFor(m => m.ProviderType, ProviderType.FE | ProviderType.Apprenticeship, new { @class = "govuk-radios__input", id = "Both" })
```
[the remark this answers is not in the document]
Comment (gunndabad) [reply src/Dfc.CourseDirectory.Web/Views/Provider/UpdateProviderType.cshtml:42]: There's another PR for replacing this page with a v2 version so I kept change here a simple as possible

[the remark this answers is not in the document]
Comment (jameslawesynergy) [reply src/Dfc.CourseDirectory.Web/Views/Provider/UpdateProviderType.cshtml:42]: Correction, works fine.

File: src/Dfc.CourseDirectory.Core/Models/ProviderType.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (jameslawesynergy) [reply src/Dfc.CourseDirectory.Core/Models/ProviderType.cs:21]: Correction, works fine.

**Pull request merged**